std/Reflect.hx

```diff
@@ -158,7 +158,7 @@ extern class Reflect {
 
 		- class instance
 		- structure
-		- `Class<T>`
+		- `Class<T>` (including `String`)
```

Comment:
Why do you feel the need to explicitly list String?

Reply:
I think this method can be more often used to detect objects for futher iteration (like jsons) or access some config arg structures, so String is special case here, since you don't want to iterate String fields.

Reply:
OKlk, I understand where you are getting at however I am unsure if this is the right place and/or way to educate people about the fact that Strings are like in most (if not all object oriented languages) objects. I think a comment out of context like this just increases confusion for everyone (the majority?) who knows about this. I for example was wondering if there is something new broken in Haxe I wasn't aware of, hence my initial question.